corehq/toggles.py

```diff
@@ -1070,6 +1072,7 @@ def _commtrackify(domain_name, toggle_is_enabled):
     "Enable custom enikshay functionality: additional user and location validation",
     TAG_ONE_OFF,
     namespaces=[NAMESPACE_DOMAIN],
+    always_enabled={"enikshay"},
```

Comment:
Actually I'd like this one dynamic at least for now, so we can do UAT on prod and turn off if it's overly restrictive.